lib/commons/dom/create-grid.js

```diff
@@ -255,26 +255,41 @@ function getStackingOrder(vNode, parentVNode) {
     vNode.getComputedStylePropertyValue('position') !== 'static';
   const floated = vNode.getComputedStylePropertyValue('float') !== 'none';
 
-  // flex and grid items can use z-index even if position: static
-  // @see https://www.w3.org/TR/css-flexbox-1/#painting
-  // @see https://www.w3.org/TR/css-grid-1/#z-order
-  if (
-    !['auto', '0'].includes(zIndex) &&
-    (positioned || isFlexOrGridContainer(parentVNode))
-  ) {
-    // if a positioned element has a z-index > 0, find the first
+  if (isStackingContext(vNode, parentVNode)) {
```

Comment:
It's because the grid creates nodes and parents for elements that aren't in the virtual tree, but never assigns parents unless it creates a new node. So if `axe.run(#fixture)`, `#fixture` will still have a null parent but for the grid the parent element is created and passed around.